connector.go

```diff
@@ -281,29 +280,37 @@ func (c *Connector) CreateObject(obj IBObject) (ref string, err error) {
 }
 
 func (c *Connector) GetObject(obj IBObject, ref string, res interface{}) (err error) {
```

Comment:
since we  are changing the method signatures, why not add options instead of just forcedProxy. 
Since we may need to include other query params , like "max_results/paging" etc.

may be like 
take another param, ...option in GetObject
`
type Options struct {
       forceProxy bool
       returnFields []string
       eaForSearch  bool
       max_results int
       paging bool
}

type option func(*Options)
`
we can build query params using the options while doing get. What say @yuewko @jkraj

Reply:
done